src/App.spec.tsx

```diff
@@ -1,18 +1,13 @@
-import { fireEvent, render, screen } from "@testing-library/react";
-import { describe, expect, it } from "vitest";
-import App from "./App";
-
 describe("App component", () => {
-  it("renders the Vite and React logos", () => {
-    render(<App />);
-    expect(screen.getByAltText("Vite logo")).toBeInTheDocument();
-    expect(screen.getByAltText("React logo")).toBeInTheDocument();
-  });
-
-  it("increments the count when the button is clicked", () => {
-    render(<App />);
-    const button = screen.getByText(/count is 0/i);
-    fireEvent.click(button);
-    expect(button).toHaveTextContent("count is 1");
-  });
+  // it("renders the Vite and React logos", () => {
+  //   render(<App />);
+  //   expect(screen.getByAltText("Vite logo")).toBeInTheDocument();
+  //   expect(screen.getByAltText("React logo")).toBeInTheDocument();
+  // });
+  // it("increments the count when the button is clicked", () => {
+  //   render(<App />);
+  //   const button = screen.getByText(/count is 0/i);
+  //   fireEvent.click(button);
+  //   expect(button).toHaveTextContent("count is 1");
+  // });
```

Comment:
Do we need this commented code? You can always get it from GIT history if you need some example

Reply:
I would definitely remove it from the codebase.